docs/stack/use-cases/_index.md

```diff
@@ -0,0 +1,417 @@
+- Who are the friends of my friend A, who are also friends with person B?
+
+**Recommendation engine (retail)**
+
+A graph database can help recommend to user A products that their friends have bought, and that currently have a margin higher than X percent and large stock in a storage close to them by answering these questions:
+
+- What are other products in a similar price range and with a similar textile as what this user usually buys?
+- What are some products that other users have bought together with the product this user just added to their cart?
+- In which store in my city can I locate this item?
+
+**Geo navigation (commerce and travel)**
+
+- List all coffee shops in an area of 2 km, with an average rating higher than 3.5.
+- List all parks around me that my friends have been to.
+
+**Operations management (networks and IT)**
+
+- Which parts of the network does this customer rely on?
+- Which customers/applications will be affected if this network element fails?
+- Is there redundancy in the network for this customer?
+
+**Fraud detection**
+
+- Fraud in health care: Are there any doctors/clinics that are prescribing an unusually high number of a certain drug?
+
+- Fraud in telecom: Does this call/SMS follow a regular calling pattern? It is a new account that just dialed an unusually high number of people, with a short average call-time and was rejected many times, so the receiver of the call should be warned by a real-time notification.
+
+- Fraud in finance: Is this transaction between sender A and recipient B valid? Has the card used by the sender been reported as stolen? Has the device that the recipient is using ever been connected to a fraudulent account?
+
+### Learn more
+
+[RedisGraph](/docs/stack/graph)
+
+## RedisTimeSeries
+
+### Patterns and use cases
+
+**Monitoring (data center)**
+
+Modern data centers have a lot of moving pieces, such as infrastructure (servers and networks) and software systems (applications and services) that need to be monitored around the clock.
+
+RedisTimeSeries allows you to plan for new resources upfront, optimize the utilization of existing resources, reconstruct the circumstances that led to outages, and identify application performance issues by analyzing and reporting on the following metrics:
+
+- Maximum CPU utilization per server
+- Maximum network latency between two services
+- Average IO bandwidth utilization of a storage system
+- 99th percentile of the response time of a specific application outages
+
+**Weather analysis (environment)**
+
+A simple use case of using RedisTimeSeries to measure environment is to track how many hours of sunshine and how much rain is observed over the last several years. Seasonally, you can measure average rain, average temperature, maximum of sunny hours, for example. Watch the increase of the maximum temperature over the years. Predict weather in a specific region for a particular week of the year. Example metrics include:
```

Comment:
wording: to measure environment

Reply:
What's your suggestion?

Reply:
RedisTimeSeries can be used to track environmental measurements such as the number of daily sunshine hours and hourly rainfall depth, over a period of many years. Seasonally, you can measure average rainfall depth, average daily temperature, and the maximum number of sunny hours per day, for example. Watch the increase of the maximum daily temperature over the years. Predict the expected temperature and rainfall depth in a specific location for a particular week of the year. 

Multiple time series can be collect, each for a different location. By utilizing secondary indexes, measurements can be aggregated over given geographical regions (e.g., minimal and maximal daily temperature in Europe) or over locations with specific attributes (e.g., average rainfall depth in mountainous regions).

Example metrics include: ...